automated install/basic-install.sh

```diff
@@ -913,6 +905,15 @@ install_dependent_packages INSTALLER_DEPS[@]
 # Install packages used by the Pi-hole
 install_dependent_packages PIHOLE_DEPS[@]
```

Comment:
The update dialog should show between the installer deps and the Pi-hole deps, because at that point we know that the installer can run, but we don't know if the user actually wants to install/update. If we add more deps, then the user would be forced to install those before they get the choice of not updating.